projects/services-ogc/src/lib/owc/types/owc-json.ts

```diff
@@ -47,18 +38,18 @@ export interface IOwsContext extends GeoJSON.FeatureCollection<GeoJSON.GeometryO
     creator?: IOwsCreator;
     /**
      * Properties of the display in use when the context document was created (for display based applications only).
-     * This class is optional and intended for creator applications that use a graphical user interface with a geographical display within a fixed pixel size and not scalable to different computational devices 
+     * This class is optional and intended for creator applications that use a graphical user interface with a geographical display within a fixed pixel size and not scalable to different computational devices
      */
     display?: IOwsCreatorDisplay[];
     /** Information about rights held in and over the Context document */
     rights?: string;
     /**
-     * Date or range of dates relevant to the resource 
+     * Date or range of dates relevant to the resource
      * time range which is expected to be of interest to the user.
      */
     date?: DateString;
     /** This array is an optional and expresses categories related to this Context document */
-    categories?: IOwsCategory[];
+    categories?: IOwsCategorie[];
```

Comment:
Here I used the wording from https://docs.opengeospatial.org/is/14-055r2/14-055r2.html#29

Renaming exported interfaces is also a breaking Change.

Reply:
Again, this is the same unwanted regression as with the disappearance of the type-checkers - happened because in our old project we had used an old version of the ukis-frontend-libraries. The old changes now overwrote the new ones. Has been corrected now!